api-reference/10 UI Widgets/GridBase/1 Configuration/rowDragging/allowReordering.md

```diff
@@ -8,13 +8,10 @@ default: false
 Allows row reordering using drag and drop gestures.
 
 ---
-This option only enables dragging. You should also implement the reordering logic in the [onReorder]({basewidgetpath}/Configuration/rowDragging/#onReorder) handler to enable row reordering inside a component. If you need to perform cross-component drag and drop, implement the dragging logic in the [onAdd]({basewidgetpath}/Configuration/rowDragging/#onAdd) handler.
+This option allows users to reorder rows only in the UI. You should also handle drag and drop in code. If users reorder rows inside a single component, implement the [onReorder]({basewidgetpath}/Configuration/rowDragging/#onReorder) handler. 
 
-#include common-demobutton-named with {
-    name: "Row Reordering",
+To configure drag and drop between components, follow the instructions from the [group]({basewidgetpath}/Configuration/rowDragging/#group) topic.
```

Comment:
Since the `group` article doesn't really provide any instructions:

```suggestion
For information on drag and drop between components, refer to the [group]({basewidgetpath}/Configuration/rowDragging/#group) topic.
```